src/markov/markov_approx.jl

```diff
@@ -73,13 +73,13 @@ function tauchen(N::Integer, ρ::Real, σ::Real, μ::Real=0.0, n_std::Integer=3)
     #       the cdf with a function that allows the distribution of input
     #       arguments to be [μ/(1 - ρ), 1] instead of [0, 1]
 
-    y .+= μ / (1 - ρ) # center process around its mean (wbar / (1 - rho))
+    yy = zeros(size(y)) .+ (μ / (1 - ρ)) # center process around its mean (wbar / (1 - rho)) in new variable
```

Comment:
@spencerlyon2 Of course, thanks for the correction. I have changed my commit; I'm a bit of a beginner here with this process so please let me know if I have missed some part of the chain of events.